packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -1,193 +1,161 @@
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieNotification } from '../../src/index.ts';
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
 import { type NotificationProps, headingLevels, variants } from '../../src/defs.ts';
 
-const rootSelector = 'pie-notification';
-const componentSelector = `[data-test-id="${rootSelector}"]`;
-const iconCloseSelector = `[data-test-id="${rootSelector}-icon-close"]`;
-const slottedIconSelector = `[data-test-id="${rootSelector}-icon-slotted"]`;
-const headingIconInfoSelector = `[data-test-id="${rootSelector}-heading-icon-info"]`;
-const headingIconSuccessSelector = `[data-test-id="${rootSelector}-heading-icon-success"]`;
-const headingIconWarningSelector = `[data-test-id="${rootSelector}-heading-icon-warning"]`;
-const headingIconErrorSelector = `[data-test-id="${rootSelector}-heading-icon-error"]`;
-const headingSelector = `[data-test-id="${rootSelector}-heading"]`;
-const footerSelector = `[data-test-id="${rootSelector}-footer"]`;
-const leadingActionSelector = `[data-test-id="${rootSelector}-leading-action"]`;
-const supportingActionSelector = `[data-test-id="${rootSelector}-supporting-action"]`;
+import { notification } from '../helpers/page-object/selectors.ts';
 
 const slotContent = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet tincidunt est, vitae vulputate turpis. Cras pretium venenatis elementum. Duis tristique neque non varius tempor. In hac habitasse platea dictumst. Aenean accumsan vehicula urna. Cras fringilla sed ipsum nec dignissim. Aliquam sit amet ullamcorper ligula.';
```

Comment:
I'd probably remove this as it's the same as the default slot value set by Storybook. Once this has been changed, simply refactor to:

`await notificationPage.load();`


```suggestion
```